pkg/repository/clone_mirror.go

```diff
@@ -52,7 +52,10 @@ func CloneMirror(repo *V1Repository,
 	targetDir string,
 	selectedVersions []string,
 	options CloneOptions) error {
-	fmt.Printf("Start to clone mirror, targetDir is %s, selectedVersions are [%s]\n", targetDir, strings.Join(selectedVersions, ","))
+	if targetDir == repo.Mirror().Source() {
```

Comment:
Should we trim suffix `/` before compare? The targetDir may be `/path/to/dir/` and the `repo.Mirror().Source()` may be `/path/to/dir`.

Reply:
Yes indeed.